### PR TITLE
COMMERCE-4697 Fixed input validators

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_definition_option_rel.jsp [hunk 1]
```diff
@@ -55,7 +55,11 @@ String defaultLanguageId = cpDefinitionOptionRelDisplayContext.getCatalogDefault
 				</div>
 
 				<div class="col-6">
-					<aui:input label="position" name="priority" />
+					<aui:input label="position" name="priority">
+						<aui:validator name="number" />
+
+						<aui:validator name="min">[0]</aui:validator>
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_definition_option_rel.jsp:61]: @Commando18 we can sort, no new line needed.
```suggestion
						<aui:validator name="min">[0]</aui:validator>
						<aui:validator name="number" />
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**